README.md

```diff
@@ -1,21 +1,43 @@
-# Paranomax Mobile App
+<p align="center">
+<img src="/assets/logo-mobile-app-beta.png" height="100"/>
+</p>
+<h1 align="center">Panoramax Mobile App</h1>
+
+Panoramax mobile is an application to capture photo sequences with your mobile and send these sequences to a Panoramax instance.
+
+> [!NOTE]
+> The app is under development, a first version is available [here](https://github.com/nobelization/panoramax-mobile-app/releases/download/v1.2.1-beta/app-release-v1_2_1-beta.apk).
+
+> You can follow progression in this [project](https://github.com/orgs/nobelization/projects/1).
+
+<div align="center">
+  <!--- <a href="https://f-droid.org/packages/app_id"> --->
+    <img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="80" align="center"/>
+  <!---</a>--->
+  <!--- <a href="https://play.google.com/store/apps/app_id"> --->
+    <img src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' alt='Get it on Google Play' height="80" align="center"/>
+  <!---</a>--->
+  <!--- <a href="https://apps.apple.com/us/app/app_id"> --->

```

Comment:
you updated the links, but don't you think we should remove/comment all those block apart from the one with the APK? (the `<a>` tag is commented, but the image is kept, leading them to be shown)

Reply:
Ok, I will try to do this tonight :)

Reply:
Done